Tests/TuistKitTests/Services/Inspect/LintRedundantImportsServiceTests.swift

```diff
@@ -0,0 +1,93 @@
+import FileSystem
+import Foundation
+import Mockable
+import MockableTest
+import Path
+import TuistCore
+import TuistLoader
+import TuistSupport
+import TuistSupportTesting
+import XcodeGraph
+import XCTest
+
+@testable import TuistKit
+
+final class LintRedundantImportsServiceTests: TuistUnitTestCase {
+    private var configLoader: MockConfigLoading!
+    private var generatorFactory: MockGeneratorFactorying!
+    private var targetScanner: MockTargetImportsScanning!
+    private var subject: InspectImportsService!
+    private var generator: MockGenerating!
+
+    override func setUp() async throws {
```

Comment:
```suggestion
    override func setUp() {
```

No need to use `async throws` afaict